### ISSUE TITLE
NVDA says space instead of the actual character when arrowing across text in the command prompt

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3476]
**Reported by camlorn on 2013-08-29 02:35**
When arrowing across entered text in the command prompt with the regular arrow keys (that is, NVDA-specific keystrokes aren't involved), NVDA is saying space instead of the actual characters.  This is particularly pronounced in the python repl, but can be seen anywhere.  I've got one other person independently verifying this at the moment.  It seems to have been introduced with today's snapshot: next-9651,359c73b.  This is bad enough that editing entered text is basically impossible.

**Comment by bhavyashah** [comment]
@camlorn Could you please perform testing as requested in https://github.com/nvaccess/nvda/issues/3476#issuecomment-155309155 by @jcsteh and revert with results?

**Comment by Adriani90** [comment]
I cannot reproduce this with NVDA 2018.2.1. on Windows 10.

**Comment by Adriani90** [comment]
it is also not reproducible with NVDA 2018.4 RC1. @ehollig I think we can close this issue and reopen it if someone encounters this issue again.

**Comment by leonardder** [comment]
This issue has been poked several times, but unfortunately we haven't got any response from @camlorn. Closing for now.
